jenkins-dsl/cassandra_pipeline.groovy

```diff
@@ -305,6 +305,17 @@ pipeline {
           }
         }
       }
+      stage('Dependency check') {
+          steps {
+              sh "ant -f ./cassandra/build.xml dependency-check"
```

Comment:
How long does this stage take? 

Let's move `ant -f ./cassandra/build.xml dependency-check` into the `build-scripts/cassandra-artifacts.sh` script. It makes sense there, and that avoids having to introduce an additional stage here and in the jenkinsfiles.